nixos/modules/security/wrappers/default.nix

```diff
@@ -188,6 +188,16 @@ in
       '';
     };
 
+    security.wrapperDirSize = lib.mkOption {
+      default = "50%";
+      example = "10G";
+      type = lib.types.str;
+      description = ''
+        Size limit for the /run/wrappers tmpfs. Look at mount(8), tmpfs size option,
+        for the accepted syntax. WARNING: don't set to less than 1GB.
```

Comment:
```
Filesystem      Size  Used Avail Use% Mounted on
tmpfs            16G  480K   16G   1% /run/wrappers

```

1GB seems like a lot. I would assume that something like 64MB is more than enough: starting with 480K, times two for `nixos-rebuild switch` (I suppose?) and an extra almost two orders of magnitude for someone's crazy use case. What am I missing?

Reply:
Agreed, 1GB is more than you'll ever need. I was being perhaps too cautious. +1 I'll reduce this warning to say 64MB, that seems like a quite reasonable estimate.